pkg/templates/templates_generated.go

```diff
@@ -5399,6 +5399,18 @@ write_files:
     #EOF
 {{- end}}
 
+{{- if ShouldConfigureCustomCATrust}}
+{{range $i, $cert := GetCustomCATrustConfigCerts}}
+{{/* adding a prefix made of zeros to match removal logic used by custom ca trust pod, which handles old cert removal */}}
+- path: /usr/local/share/ca-certificates/00000000000000cert{{$i}}.crt
+  permissions: "0644"
+  owner: root
+  content: |
+    {{$cert}}
+  #EOF
```

Comment:
I think you do actually want the EOF indented, otherwise it's going to count as a yaml key not part of multiline string

good thing to test e2e :)